packaging/linux/post_install.sh

```diff
@@ -43,88 +24,6 @@ make_mountpoint() {
   fi
 }
 
-systemd_exec_as() {
-    user=$1
-    shift
-
-    # Keybase supports running with and without systemd. In order for `su`
-    # to work with systemd commands, we need to pass in XDG_RUNTIME_DIR.
-    # /run/user/{uid} is the default when using systemd. We can't know the
-    # true value without installing systemd itself, which we don't want to
-    # depend on, so we use this default. Users who want to use a custom
-    # XDG_RUNTIME_DIR should specify it in their .profile so it is known
-    # by su -, which doesn't load pam_systemd. Unfortunately, this also
-    # results in an extra su invocation.
-    user_xdg_runtime_dir=""
-    # shellcheck disable=SC2016
-    # Intentionally do not expand $XDG_RUNTIME_DIR; we want the user's shell to expand it
-    if ! user_xdg_runtime_dir="$(su --login "$user" -c 'echo $XDG_RUNTIME_DIR')" || [ -z "$user_xdg_runtime_dir" ]; then
-        user_xdg_runtime_dir="/run/user/$(id -u "$user")" || return 1
-    fi
-
-    # To support restarting without systemd, we'd also have to pass in DISPLAY,
-    # but there's no easy way to figure that out.
-    su --login "$user" -c "XDG_RUNTIME_DIR=$user_xdg_runtime_dir $* 2> /dev/null"
-}
-
-# Exits with 0 iff the given user is running the service with systemd
-systemd_unit_active_for() {
-    user=$1
-    service=$2
-    command -v systemctl &> /dev/null && systemd_exec_as "$user" "systemctl --user -q is-active $service"
-}
-
-safe_restart_systemd_services() {
```

Comment:
I'm getting rid of this. I don't think there's currently a reliable way to do this across platforms and for users who set environment variables in shell rc files. And also while preserving $DISPLAY. So it's just going to ask user to restart and then the CLI will warn them again if they still need to. Later we can add to gui.